### PR TITLE
Release 1.0.0 rc.1

### PR DESCRIPTION
**Pull request by fcgravalos**
This will keep track of any fixes before we get into stable 1.0.0

### CODE REVIEW

**Comment by fcgravalos** [comment]
Well, release candidates are all in the form `rc.X`. I should have probably done the same for the snapshot ones.

**Comment by fcgravalos** [comment]
We've been running snapshot versions for a while, so I'm merging this and I'll create a release version. If we have to do more fixes we'll create rc.2 and push them too

File: config/default/manager_image_patch.yaml [hunk 1]
```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: registry.hub.docker.com/secrets-manager:v1.0.0-snapshot-1
+      - image: docker.tuenti.io/sre/secrets-manager:v1.0.0-rc.1
```
Comment (dannyk81) [inline config/default/manager_image_patch.yaml]: this is a private registry, not sure that should be here.

Comment (fcgravalos) [reply config/default/manager_image_patch.yaml]: right this is due to some tests I did and I forget to remove it. Thx.

**Pull request merged**